src/shared/utils/state.ts

```diff
@@ -51,6 +52,38 @@ export const getFulfilledData = <T extends Record<string, BaseSchema>, K>({
     }
   });
 
+export const getFulfilledDataWithConcatenatedResult = <T extends Record<string, BaseSchema>, K>({
+  builder,
+  thunk,
+  key,
+  initialState,
+  mapper = (o) => o,
```

Comment:
```suggestion
  mapper = (data) => data,
```
also pls change in getFulfilledData